src/seedu/addressbook/data/Tagging.java

```diff
@@ -0,0 +1,36 @@
+package seedu.addressbook.data;
+
+import seedu.addressbook.data.person.Name;
+import seedu.addressbook.data.tag.Tag;
+
+public class Tagging {
+
+	private enum Action{
+		ADD, REMOVE;
```

Comment:
Note that our indentation style uses 4 spaces instead of tabs, the steps to change to spaces can be seen here http://stackoverflow.com/questions/407929/how-do-i-change-eclipse-to-use-spaces-instead-of-tabs